static/js/redux/ui/Semesterly.tsx

```diff
@@ -246,7 +246,7 @@ const Semesterly = () => {
 
 const SemesterlyWithErrorBoundary = withErrorBoundary(Semesterly, {
   FallbackComponent: FallBack,
-  onError(error, info) {
+  onError(error: any, info: any) {
```

Comment:
@JiaqiWang18 is this right? i just changed this since tsx was complaining about them having an any type

Reply:
this shouldn't be necessary because typescript will automatically deduce the type, and we should avoid using `any` in TS 
![image](https://github.com/jhuopensource/semesterly/assets/37493948/cb217847-5dea-4588-a8b5-e8ca1a7afca7)

Reply:
if it is still complaining, try running `npm i` OUTSIDE of the docker environment and reload the window because VScode uses the local `node_modules` for type hints and the package might be missing, causing the complain.
Just ignore it if that doesn't work.